RunGoogleClassroom.py

```diff
@@ -74,28 +74,30 @@ def Verify_Assignment(service, assignmentName):
             print("Assignment: " + assignmentName + " Not found, do not specify the assignment on the command line to get a full list")
     return assignmentSelected
 
-CHUNK_SIZE = 1024 * 1024
-def Copy_Student_Java_Files(tempDir, service, drive, assignmentSelected, excelWriter, addOutput, addFile, goldLines, filter1):
-    RunJavaUtils.Create_Header( excelWriter, addOutput, addFile, goldLines)
-    studentSubmissions = service.courses().courseWork().studentSubmissions()
-    submissions_for_sslected_assignment = studentSubmissions.list(courseId=courseId,
-                                                                  courseWorkId=assignmentSelected['id']).execute()
-
-
+def Sort_Student_Names(studentInfo):
     # We want to print the submissions sorted by name, so create a sorted list of names
     # and a map of ID -> name
     studentIDToNameMap={}
-    studentNameList=[]
-    studentInfo = service.courses().students().list(courseId=courseId).execute();
+    studentNameList=[] 
     studentIDs = studentInfo['students']
     for student in studentIDs:
-        studentName = student['profile']['name']['fullName'].lower()
+        studentName = student['profile']['name']['familyName'].lower()
         studentNameList.append(studentName)
         studentID = student['profile']['id']
         studentIDToNameMap[studentID]=studentName
-
     #sort the student names
     studentNameList.sort()
+    return (studentNameList, studentIDToNameMap)
+
+CHUNK_SIZE = 1024 * 1024
```

Comment:
unused?

Reply:
It is used in the Copy_Student_Java_Files function.  I just added a space between the function & return